### ISSUE TITLE
Unable to download from git repo

### ISSUE THREAD

**Issue by paradox41** [kennethreitz/pipenv#129]
Copied from #116 

I am now unable to fetch packages from public git repos. It still tries to fetch from PyPI

#### Traceback

```
pipenv install git+https://github.com/paradox41/heroprotocol.git#egg=heroprotocol 
Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 594, in install
    ensure_project(three=three)
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 92, in ensure_project
    ensure_pipfile()
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 69, in ensure_pipfile
    ensure_proper_casing()
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 128, in ensure_proper_casing
    casing_changed = proper_case_section('packages')
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 111, in proper_case_section
    new_casing = proper_case(dep)
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 492, in proper_case
    raise IOError('Unable to find package {0} in PyPI repository.'.format(crayons.green(package_name)))
IOError: Unable to find package heroprotocol in PyPI repository.
```

#### Pipfile
```
[[source]]
url = "https://pypi.python.org/simple"
verify_ssl = true

[packages]
flask-oauthlib = "==0.9.3"
py = "==1.4.32"
"delegator.py" = "==0.0.7"
configargparse = "==0.11.0"
pytest-cov = "==2.2.0"
graphene = "==1.1.3"
fixture = "==1.5.9"
dateutils = "==0.6.6"
ipython-genutils = "==0.1.0"
packaging = "==16.8"
ipython = "==5.1.0"
pydash = "==3.4.8"
pygments = "==2.2.0"
promise = "==1.0.1"
prompt-toolkit = "==1.0.9"
pyparsing = "==2.1.10"
click = "==6.7"
appdirs = "==1.4.0"
mako = "==1.0.6"
oauthlib = "==2.0.1"
pexpect = "==4.2.1"
amqp = "==2.1.4"
flask = "==0.12"
ipdb = "==0.10.1"
ujson = "==1.35"
flask-sqlalchemy = "==2.1"
redis = "==2.10.5"
ptyprocess = "==0.5.1"
kafka-python = "==1.3.2"
celery = "==4.0.2"
graphql-relay = "==0.4.5"
pytest = "==2.8.7"
flask-bcrypt = "==0.7.1"
kombu = "==4.0.2"
typing = "==3.5.3.0"
toml = "==0.9.2"
itsdangerous = "==0.24"
bcrypt = "==3.1.2"
uwsgi = "==2.0.14"
python-editor = "==1.0.3"
virtualenv = "==15.1.0"
crayons = "==0.1.2"
requirements-parser = "==0.1.0"
pathlib2 = "==2.2.1"
simplejson = "==3.10.0"
vine = "==1.1.3"
flask-login = "==0.4.0"
graphene-sqlalchemy = "==1.1.1"
markupsafe = "==0.23"
billiard = "==3.5.0.2"
colorama = "==0.3.7"
coverage = "==3.7.1"
flask-shell-ipython = "==0.2.2"
python-dateutil = "==2.6.0"
coveralls = "==1.1"
iso8601 = "==0.1.11"
pickleshare = "==0.7.4"
six = "==1.10.0"
decorator = "==4.0.11"
"backports.shutil-get-terminal-size" = "==1.0.0"
mpyq = "==0.2.5"
flask-script = "==2.0.5"
flask-graphql = "==1.4.0"
graphql-core = "==1.0.1"
pipenv = "==0.2.8"
enum34 = "==1.1.6"
pytz = "==2016.10"
sqlalchemy = "==1.1.5"
wcwidth = "==0.1.7"
jinja2 = "==2.9.4"
traitlets = "==4.3.1"
requests-oauthlib = "==0.7.0"
alembic = "==0.8.10"
pycparser = "==2.17"
psycopg2 = "==2.6.2"
singledispatch = "==3.4.0.3"
requests = "==2.13.0"
docopt = "==0.6.2"
simplegeneric = "==0.8.1"
cffi = "==1.9.1"
scandir = "==1.4"

[packages.heroprotocol]
git = "https://github.com/paradox41/heroprotocol.git"

[packages.resource_alchemy]
git = "https://github.com/tomneyland/resource-alchemy.git"
```

#### Pipefile.lock

```
{
    "default": {
        "Mako": "==1.0.6",
        "flask-oauthlib": "==0.9.3",
        "flask-sqlalchemy": "==2.1",
        "configargparse": "==0.11.0",
        "Flask": "==0.12",
        "cffi": "==1.9.1",
        "ipython-genutils": "==0.1.0",
        "packaging": "==16.8",
        "pygments": "==2.2.0",
        "promise": "==1.0.1",
        "Werkzeug": "==0.11.15",
        "prompt-toolkit": "==1.0.9",
        "pyparsing": "==2.1.10",
        "appdirs": "==1.4.0",
        "oauthlib": "==2.0.1",
        "Flask-Script": "==2.0.5",
        "pexpect": "==4.2.1",
        "flask": "==0.12",
        "ipdb": "==0.10.1",
        "py": "==1.4.32",
        "celery": "==4.0.2",
        "Jinja2": "==2.9.4",
        "python-editor": "==1.0.3",
        "typing": "==3.5.3.0",
        "itsdangerous": "==0.24",
        "appnope": "==0.1.0",
        "sqlalchemy": "==1.1.5",
        "pathlib2": "==2.2.1",
        "simplejson": "==3.10.0",
        "vine": "==1.1.3",
        "Flask-OAuthlib": "==0.9.3",
        "iso8601": "==0.1.11",
        "markupsafe": "==0.23",
        "billiard": "==3.5.0.2",
        "jinja2": "==2.9.4",
        "python-dateutil": "==2.6.0",
        "flask-graphql": "==1.4.0",
        "decorator": "==4.0.11",
        "graphql-core": "==1.0.1",
        "mpyq": "==0.2.5",
        "flask-script": "==2.0.5",
        "coveralls": "==1.1",
        "delegator.py": "==0.0.7",
        "Pygments": "==2.2.0",
        "requests-oauthlib": "==0.7.0",
        "uWSGI": "==2.0.14",
        "pycparser": "==2.17",
        "pydash": "==3.4.8",
        "resource-alchemy": "==0.1.0",
        "dateutils": "==0.6.6",
        "resource_alchemy": {
            "git": "https://github.com/tomneyland/resource-alchemy.git"
        },
        "graphene": "==1.1.3",
        "Flask-SQLAlchemy": "==2.1",
        "fixture": "==1.5.9",
        "SQLAlchemy": "==1.1.5",
        "singledispatch": "==3.4.0.3",
        "click": "==6.7",
        "mako": "==1.0.6",
        "amqp": "==2.1.4",
        "ujson": "==1.35",
        "six": "==1.10.0",
        "redis": "==2.10.5",
        "ptyprocess": "==0.5.1",
        "kafka-python": "==1.3.2",
        "graphql-relay": "==0.4.5",
        "pytest": "==2.8.7",
        "flask-bcrypt": "==0.7.1",
        "kombu": "==4.0.2",
        "Flask-Bcrypt": "==0.7.1",
        "bcrypt": "==3.1.2",
        "uwsgi": "==2.0.14",
        "heroprotocol": {
            "git": "https://github.com/paradox41/heroprotocol.git"
        },
        "virtualenv": "==15.1.0",
        "crayons": "==0.1.2",
        "requirements-parser": "==0.1.0",
        "traitlets": "==4.3.1",
        "flask-login": "==0.4.0",
        "graphene-sqlalchemy": "==1.1.1",
        "colorama": "==0.3.7",
        "coverage": "==3.7.1",
        "flask-shell-ipython": "==0.2.2",
        "ipython": "==5.1.0",
        "pickleshare": "==0.7.4",
        "Flask-Login": "==0.4.0",
        "Flask-GraphQL": "==1.4.0",
        "ConfigArgParse": "==0.11.0",
        "pipenv": "==0.2.8",
        "MarkupSafe": "==0.23",
        "enum34": "==1.1.6",
        "simplegeneric": "==0.8.1",
        "pytest-cov": "==2.2.0",
        "wcwidth": "==0.1.7",
        "pytz": "==2016.10",
        "toml": "==0.9.2",
        "psycopg2": "==2.6.2",
        "requests": "==2.13.0",
        "docopt": "==0.6.2",
        "backports.shutil-get-terminal-size": "==1.0.0",
        "alembic": "==0.8.10",
        "scandir": "==1.4"
    },
    "develop": {},
    "_meta": {
        "sources": [
            {
                "url": "https://pypi.python.org/simple",
                "verify_ssl": true
            }
        ],
        "requires": {},
        "Pipfile-sha256": "16d57a0d06ac907d923c9b516310698779fab27ca9487b89de1c26d09fd1bd9e"
    }
}
```

This was working prior to v3.0.0. I just tried installing again today, and it threw out this error.

**Comment by nateprewitt** [comment]
This should be addressed with #131. Please reopen this is you are still encountering this issue when 3.2.5 is released.

Thanks for letting us know!

**Comment by paradox41** [comment]
This issue looks to have been fixed. Thanks for the quick turnaround!
